wireshark/wolfssl-qsc.h

```diff
@@ -31,6 +37,12 @@
     { 12094, "p256_kyber_90s_level1" }, \
     { 12095, "p384_kyber_90s_level3" }, \
     { 12096, "p521_kyber_90s_level5" }, \
-    { 0xfe0b, "falcon_level1" },\
-    { 0xfe0e, "falcon_level5" },
+    { 0xfe0b, "falcon_level1" }, \
+    { 0xfe0e, "falcon_level5" }, \
+    { 0xfea0, "dilithium_level2" }, \
```

Comment:
SIGs under the "QSC_KEMS" define seem "weird": Is this intentional?

Reply:
Yes, I would agree, but this was the convention that you created.  (Falcon was already there in PR 105 which I believe you authored).  Probably better names would be `QSC_CODEPOINTS` and `QSC_OIDS` but that is out of scope of this change.

Reply:
Hi @baentsch , 

Sorry for the delayed response on the comment below, had to go eat lunch.  

I prefer you to handle the OQS side.  The reason being you decide your naming conventions and which names are to be displayed or left unknown for your flavour.